### ISSUE TITLE
Re-word or deprioritize "Legacy Programs" card

### ISSUE THREAD

**Issue by emilyville** [department-of-veterans-affairs/vets-website#1088]
Feedback from a few different types of usability sessions and card sorting has shown that "legacy programs" is not a term people use consistently like we mean it here. Can we reword this so it's more user friendly and also potentially reprioritize on this page? 

![screen shot 2016-01-27 at 12 35 33 pm](https://cloud.githubusercontent.com/assets/1307774/12627413/86550670-c4f2-11e5-98f8-371f366d9fcc.png)

tagging for you all to weigh in - 
@rcarthur @dannychapman @koconnor719 @maryannbrody

**Comment by dannychapman** [comment]
I think the IA here should be reworked in general - there's a lot of "underwear on the outside here" in terms of organization. For example - can we put the GI Bill Comparison tool inside the GI Bill section? Perhaps legacy programs could be split up and the content integrated into other sections?

**Comment by emilyville** [comment]
Totally agree. I think the "legacy programs"  is particularly problematic because legacy means something different to some folks (what you leave behind) and here we mean "old thing we still support", no?

So yes to both but a new card title seems most important until we start to really tacky tackle sitewide ia! 

I think we're due for a list of major links within cards - like gi tool. Since (I think) many users go directly to tools. We should validate this with testing and data, though. @maryannbrody?

**Comment by maryannbrody** [comment]
@emilyville & @dannychapman -- Also agree the IA needs to be re-worked. We are definitely working on it. We had a finding from the last round of usability testing that users go to GI Bill card for a task designed to get them to the comparison tool. I recommended combining the two and then making it easier to get to GIBCT. 

What if we just change "Legacy Programs" to ...

Title: "Other Education Benefits" 
Subtext: "You may qualify for VA's smaller (non-GI Bill )  education benefit programs." Subtext option 2: "Learn about VA's smaller (non-GI Bill ) education benefit programs."

Suggest also de-prioritizing and making it the last card on the page.

**Comment by dannychapman** [comment]
That sounds good! @maryannbrody are you working with the content team on this?

**Comment by koconnor719** [comment]
content lead for GI tool is @mollyblake

**Comment by maryannbrody** [comment]
@mollyblake if this looks good to you, can you make this change?

**Comment by mollyblake** [comment]
@maryannbrody How about "Other Educational Assistance Programs"

**Comment by bethpotts** [comment]
@dannychapman Can you help make the necessary breadcrumb changes?

**Comment by maryannbrody** [comment]
@mollyblake "Other Educational Assistance Programs" is good with me -- I would only point out that we use "education" elsewhere, so "education" might be better for consistency.

**Comment by webinista** [comment]
@emilyville I think this is already complete. That [page](https://www.vets.gov/education/) already says "Other Educational Assistance Programs." I'm going to close this. Re-open if I got it wrong.
